cmd/config.go

```diff
@@ -63,17 +80,13 @@ func validateConfig(cmd *cobra.Command, args []string) error {
 		return response.ToError()
 	}
 
-	Logger.Infof("Config file at %s is valid", configPath)
+	Logger.Infof("Config file at %s is valid", path)
 	return nil
 }
 
 func expandConfig(cmd *cobra.Command, args []string) error {
 	ctx := context.Background()
```

Comment:
Is there a reason why `expand` has different config search behaviour than `validate`? - Seems like something that will be confusing.

Reply:
Validate is treated different for compatibility reasons, those should be documented in the comments -- but if that's not clear then we should fix it.